lib/trie/lookup.go

```diff
@@ -0,0 +1,87 @@
+package trie
+
+import (
+	"bytes"
+	"errors"
+
+	"github.com/ChainSafe/chaindb"
+)
+
+var (
+	// ErrProofNodeNotFound when a needed proof node is not in the database
+	ErrProofNodeNotFound = errors.New("cannot found the state root on storage")
```

Comment:
nit: maybe should be "in storage" instead of  "on storage"

Reply:
updated!